### PR TITLE
Update what-are-chaoss-goals.md

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

A few minor updates for 2020.

### CODE REVIEW

File: About/About/what-are-chaoss-goals.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-The project aims to:
+The project goals are to:
 
-- Establish standard implementation-agnostic metrics for measuring community activity, contributions, and health
+- Establish standard implementation-agnostic metrics for measuring community health
 
 - Produce integrated open source software for analyzing software community development
 
-- Build reproducible project health reports/containers
+- Develop program for the deployment of metrics not attainable through online trace data
```
Comment (GeorgLink) [inline About/About/what-are-chaoss-goals.md:9]: "Develop program" seems to be missing something

how about: 
```suggestion
- Develop a program for the deployment of metrics not attainable through online trace data
```

Comment (germonprez) [reply About/About/what-are-chaoss-goals.md:9]: how about programs? I think I just forget an s

Comment (GeorgLink) [reply About/About/what-are-chaoss-goals.md:9]: Yes, that works too :)

**Comment by germonprez** [comment]
Could someone merge this :)

**Pull request merged**